### DISCUSSION TITLE
Maintainers wanted

### DISCUSSION THREAD

**Discussion by LordMike** [DiscUtils/DiscUtils#269] [Announcements]
@qmfrederik I'm finding it very hard to find time or energy to focus on DiscUtils. Do you know anyone who would want to chip in?
We might be able to ask on the PR's submitted, to see if anyone is willing?

(I couldn't find a mail for you)

**Comment by Olof-Lagerkvist** [comment]
I understand. I spend a lot of time on my fork of it because it is nowadays actively developed as part of a couple of different projects I am working on. So I have time and motivation to spend on it in that way. The primary demands that have driven that progress have been adding some missing features like registry transaction logs, platform independent security descriptors etc. But also efficiency in general, like avoiding temporary heap allocations and async/await patterns etc.

But since the relationship between this repo and the fork has became quite complex over the years because of lots of breaking changes etc, it would anyway be a bit complicated to work on both repos actively. Although I find it useful to keep this repo pretty much the way it is from an API compatibility perspective and only add missing stuff and fix bugs etc. So, I can most probably step in and help, but not sure about how much.
